include/glow/Graph/Graph.h

```diff
@@ -181,6 +181,9 @@ class Module final {
   /// Erase a function \p F from the module.
   void eraseFunction(Function *F);
 
+  /// \Returns the size in bytes of data used by constants.
+  uint64_t dataSize();
```

Comment:
would be clear to specify that it's for constants. maybe constantsSize() or something.